api/src/main/java/com/nicknathanjustin/streamercontracts/votes/VoteServiceImpl.java

```diff
@@ -22,7 +22,10 @@
     public void recordVote(@NonNull final UserModel voter,
                            @NonNull final ContractModel contractModel,
                            final boolean flaggedCompleted) {
-        assertContractIsVotable(contractModel);
+    	if (isContractPastSettleTimestamp(contractModel)) {
```

Comment:
Is this check better placed in the assertUserCanVoteOnContract() method?

Reply:
yea good call